### ISSUE TITLE
RFC: Requesting for Landslide Impact Function

### ISSUE THREAD

**Issue by adissadis** [inasafe/inasafe#1219]
# Problem

Landslide are common hazard type that some people ask. We need categorized impact function to address these following hazard types.
# Duration

TBA
# Current state

Draft
# Proposers

@adissadis
@mahardika
@cchristelis 
# Detail

Some people in training ask if there's any impact function for landslide. They usually use flood impact function to analyze the landslide hazard. There's document for creating landslide vulnerability area (http://tinyurl.com/landslide-doc) and the university that studying for creating landslide hazard map is UGM - Hazard Research Center. The landslide map is in province level which is general scale. 
# Proposed Solution

Create some agreement with UGM, PVMBG to create landslide susceptibility map in detail scale. Exposure data should be against road, structure and population (not raster population data but maybe we can put number of people in building for this case). 
Proposed action checklist for landslide:
1. Which road blocked by landslide?
2. If blocked, do we need some equipment to dig the landslide?
3. How do we evacuate if the evacuation roads are blocked by landslide? 
# Exposure needed

This impact function will analyze road, structure and also population
# RFC Provenace

This RFC was discussed at the InaSAFE workshop in Jakarta, Wednesday 24 September 2014. The following people were in Attendance:
- Christian Christelis
- Ranie Dwi Anugrah
- Maning
- Elida Nurokhmah
- Dewi Sulistioningrum
- Mahardika Fadmastuti
- Harry Mahardhika
- Dino
- Intan
- Vasanthi
- Wulansari Khairunisa
- Adityo Dwijananto
# Record of Votes
- 1 Adityo Dwijananto
- 1 Mahardika Fadmastuti

**Comment by timlinux** [comment]
It seems we could just be using generic IF for this. In the paper cited above they work on high/medium/low hazard zones. I think the actual hazard zone calculations described in that paper are beyond the scope of what we can should in InaSAFE. Perhaps we should focus on custom hazard classes (so we can use landslide specific wording) and hazard specific actions and notes for now.

It we are going to make it no support population rasters, but rather PIB then we need to revive the PIB work on the roadmap at some point..... I am just going to leave this parked here for now until we get actual 'GO' to work on this.

**Comment by Charlotte-Morgan** [comment]
Yes - generic is fine for this. Originally we had use cases for landslide analysis from Pak Trias and GeoRisk but this is not a priority for the DMI funded work. However, this would be well regarded as a topic for training new developers how to write additional code as it has wide application assuming inputs are available from Landslide hazard experts

**Comment by timlinux** [comment]
OK thanks for this @Charlotte-Morgan I am going to close this out and we can use landslide as the basis for training materials for advanced users / beginner developers.
